<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forest Fire Detector - Ertuğrul Kuşva</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>tree.png</normaloff>tree.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="sekmeler">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1331</width>
      <height>921</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(47, 147, 35)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="algilama_sekmesi">
     <attribute name="title">
      <string>Algılama</string>
     </attribute>
     <widget class="QPushButton" name="algilama_butonu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>810</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Algıla</string>
      </property>
     </widget>
     <widget class="QLabel" name="resim_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid lightgray;
</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="resim_label2">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid lightgray</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="konsol">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>630</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mail_butonu">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>810</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Mail Gönder</string>
      </property>
     </widget>
     <widget class="QLabel" name="resim_label3">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>320</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid lightgray</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sms_butonu">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>810</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>SMS Gönder</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ayarlar_sekmesi">
     <attribute name="title">
      <string>Ayarlar</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Yetkili adı:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Yetkili cep telefonu:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Yetkili mail adresi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="yetkili_kaydet_butonu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="konsol_ayarlar">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>630</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="yetkili_adi_text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="yetkili_cep_telefonu_text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="yetkili_mail_adresi_text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="yetkili_duzenle_butonu">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Düzenle</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="yetkili_table_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>531</width>
        <height>341</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ad_soyad</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ceptel_no</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>mail_adres</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="bolgeler_sekmesi">
     <attribute name="title">
      <string>Bölgeler</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1338</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
